app/test_main.py

```diff
@@ -1,3 +1,26 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,expected_result",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (27, 28, [2, 2]),
```

Comment:
The test case `(27, 28, [2, 2])` might not be necessary as it repeats the expected result of the previous test case `(27, 27, [2, 2])`. Consider removing it to avoid redundancy, as per the checklist item: 'Make sure that your tests don't test the same case few times.'